style.css

```diff
@@ -134,6 +132,29 @@ article{
 .hover p{
 	padding-bottom: 10px;
 }
+
+.contact-info{
+	display: flex;
+	justify-content: center;
+}
+.info{
+	margin-left: 80px;
```

Comment:
Couldn't you use a container class to horizontally align everything instead of this solution?